README.md

```diff
@@ -49,23 +182,22 @@ Tracer createWavefrontTracer(String application, String service) throws IOExcept
 
 Application tags determine the metadata (span tags) that are included with every span reported to Wavefront. These tags enable you to filter and query trace data in Wavefront.
 
-You encapsulate application tags in an `ApplicationTags` object.
+You encapsulate application tags in an `ApplicationTags` object, which is imported from the Wavefront Java SDK.
 See [Instantiating ApplicationTags](https://github.com/wavefrontHQ/wavefront-sdk-java/blob/master/docs/apptags.md) for details.
 
 ### 2. Set Up a WavefrontSender
 
-A `WavefrontSender` object implements the low-level interface for sending data to Wavefront. You can choose to send data using either the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
+A `WavefrontSender` object implements the low-level interface for sending data to Wavefront, which is imported from the Wavefront Java SDK. You can choose to send data using the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
```

Comment:
Does this sound ok (technically)?
I want to imply that we get it from the Wavefront Java SDK. Because afterward, we direct the users to the docs in the Wavefront Java SDK.